sdk-base/src/main/java/com/mapbox/maps/extension/style/StyleInterface.kt

```diff
@@ -15,6 +15,18 @@ interface StyleInterface : StyleManagerInterface {
    */
   val pixelRatio: Float
 
+  /**
+   * Whether the Style instance is valid.
+   *
+   * Style becomes invalid after MapView.onDestroy() is invoked,
+   * calling any method then could result in undefined behaviour and will print an error log.
+   *
+   * Keeping the reference to an invalid Style instance introduces significant native memory leak.
```

Comment:
Pls also add similar comments to the MapView.getMapboxMap / MapView.getStyle

Reply:
👍